resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/submit/submitsteps/DriverServiceBootstrapStepSuite.scala

```diff
@@ -52,92 +52,92 @@ private[spark] class DriverAddressConfigurationStepSuite
+      DEFAULT_DRIVER_PORT,
+      DEFAULT_BLOCKMANAGER_PORT,
+      s"$SHORT_RESOURCE_NAME_PREFIX${DriverServiceBootstrapStep.DRIVER_SVC_POSTFIX}",
+      resolvedDriverSpec.otherKubernetesResources.head.asInstanceOf[Service])
     assert(resolvedDriverSpec.driverSparkConf.get("spark.driver.port") ===
-        DEFAULT_DRIVER_PORT.toString)
+      DEFAULT_DRIVER_PORT.toString)
     assert(resolvedDriverSpec.driverSparkConf.get(
-        org.apache.spark.internal.config.DRIVER_BLOCK_MANAGER_PORT) === DEFAULT_BLOCKMANAGER_PORT)
+      org.apache.spark.internal.config.DRIVER_BLOCK_MANAGER_PORT) === DEFAULT_BLOCKMANAGER_PORT)
   }
 
   test("Long prefixes should switch to using a generated name.") {
-    val configurationStep = new DriverAddressConfigurationStep(
-        LONG_RESOURCE_NAME_PREFIX,
-        DRIVER_LABELS,
-        sparkConf.set(KUBERNETES_NAMESPACE, "my-namespace"),
-        clock)
+    val configurationStep = new DriverServiceBootstrapStep(
+      LONG_RESOURCE_NAME_PREFIX,
+      DRIVER_LABELS,
+      sparkConf.set(KUBERNETES_NAMESPACE, "my-namespace"),
+      clock)
     when(clock.getTimeMillis()).thenReturn(10000)
     val baseDriverSpec = KubernetesDriverSpec.initialSpec(sparkConf.clone())
     val resolvedDriverSpec = configurationStep.configureDriver(baseDriverSpec)
     val driverService = resolvedDriverSpec.otherKubernetesResources.head.asInstanceOf[Service]
-    val expectedServiceName = s"spark-10000${DriverAddressConfigurationStep.DRIVER_SVC_POSTFIX}"
+    val expectedServiceName = s"spark-10000${DriverServiceBootstrapStep.DRIVER_SVC_POSTFIX}"
     assert(driverService.getMetadata.getName === expectedServiceName)
     val expectedHostName = s"$expectedServiceName.my-namespace.svc.cluster.local"
     verifySparkConfHostNames(resolvedDriverSpec.driverSparkConf, expectedHostName)
   }
 
   test("Disallow bind address and driver host to be set explicitly.") {
-    val configurationStep = new DriverAddressConfigurationStep(
-        LONG_RESOURCE_NAME_PREFIX,
-        DRIVER_LABELS,
-        sparkConf.set(org.apache.spark.internal.config.DRIVER_BIND_ADDRESS, "host"),
-        clock)
+    val configurationStep = new DriverServiceBootstrapStep(
+      LONG_RESOURCE_NAME_PREFIX,
+      DRIVER_LABELS,
+      sparkConf.set(org.apache.spark.internal.config.DRIVER_BIND_ADDRESS, "host"),
+      clock)
     try {
       configurationStep.configureDriver(KubernetesDriverSpec.initialSpec(sparkConf))
       fail("The driver bind address should not be allowed.")
     } catch {
       case e: Throwable =>
         assert(e.getMessage ===
-            s"requirement failed: ${DriverAddressConfigurationStep.DRIVER_BIND_ADDRESS_KEY} is" +
-                s" not supported in Kubernetes mode, as the driver's hostname will be managed via" +
-                s" a Kubernetes service.")
+          s"requirement failed: ${DriverServiceBootstrapStep.DRIVER_BIND_ADDRESS_KEY} is" +
```

Comment:
Error message should probably say "the driver's bind address will be managed...". It's also not by a Kubernetes service anymore, but by the pod's IP address.

Reply:
Done.